app/src/wavevr/cpp/DeviceDelegateWaveVR.cpp

```diff
@@ -282,7 +282,7 @@ struct DeviceDelegateWaveVR::State {
 
       if (touchpadTouched) {
         WVR_Axis_t axis = WVR_GetInputAnalogAxis(controller.type, WVR_InputId_Alias1_Touchpad);
-        delegate->SetTouchPosition(controller.index, axis.x, -axis.y);
+        delegate->SetTouchPosition(controller.index, (axis.x + 1) * 0.5, (-axis.y + 1) * 0.5);
```

Comment:
Please add a comment on why this adjustment is being made (i.e., we are matching the Oculus touch pad range).